pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -42,6 +42,7 @@
     "System.Threading.ThreadPool": "4.3.0",
     "Microsoft.DiaSymReader.Native": "1.4.1",
     "Libuv": "1.9.1",
+    "runtime.native.System.Security.Cryptography.Apple": "4.3.1-servicing-25511-01",
```

Comment:
Those look like the slipped in after our last release. They should be marked as stable. Can you please update them to remove the pre-release identifier for those. Essentially revert https://github.com/dotnet/core-setup/commit/225035b171bfa82816be22d9d5f7364e7e34d375.